workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/EntityGameObjectLinker.cs

```diff
@@ -76,6 +95,17 @@ public void LinkGameObjectToEntity(GameObject gameObject, Entity entity, ViewCom
             spatialOSComponent.SpatialEntityId = spatialEntityId;
         }
 
+        /// <summary>
+        ///     Un-links a GameObject and ECS Entity.
```

Comment:
I think unlink is one word. Check with dictionary though.

Reply:
It is - although Rider seems to think otherwise